FluentUI.Demo/src/main/java/com/microsoft/fluentuidemo/V2DesignTokens.kt

```diff
@@ -0,0 +1,387 @@
+
+enum class Tokens {
+    GlobalTokens,
+    AliasTokens
+}
+
+class V2DesignTokens : V2DemoActivity() {
+    private fun previewShadowToken(selectedToken: FluentGlobalTokens.ShadowTokens): BasicCardTokens {
+        return object : BasicCardTokens() {
+            @Composable
+            override fun elevation(basicCardInfo: BasicCardInfo): Dp {
+                return when (basicCardInfo.cardType) {
+                    CardType.Elevated -> elevation(selectedToken)
+                    CardType.Outlined -> 0.dp
+                }
+            }
+        }
+    }
+
+    private fun previewCornerRadiusToken(selectedToken: FluentGlobalTokens.CornerRadiusTokens): BasicCardTokens {
+        return object : BasicCardTokens() {
+            @Composable
+            override fun cornerRadius(basicCardInfo: BasicCardInfo): Dp {
+                return cornerRadius(selectedToken)
+            }
+        }
+
+    }
+
+    private fun previewStrokeWidthToken(selectedToken: FluentGlobalTokens.StrokeWidthTokens): BasicCardTokens {
+        return object : BasicCardTokens() {
+            @Composable
+            override fun borderStrokeWidth(basicCardInfo: BasicCardInfo): Dp {
+                return strokeWidth(selectedToken)
+            }
+        }
+    }
+
+    @Composable
+    fun PreviewToken(tokenName: String, tokensList: Array<out Enum<*>>) {
+        val tabsList = mutableListOf<PillMetaData>()
+        var selectedToken by remember { mutableStateOf(0) }
+        tokensList.forEach {
+            tabsList.add(
+                PillMetaData(
+                    text = "$it",
+                    onClick = { selectedToken = tokensList.indexOf(it) }
+                )
+            )
+        }
+
+        Column(horizontalAlignment = Alignment.CenterHorizontally) {
+            BasicCard(
+                modifier = Modifier
+                    .size(width = 180.dp, height = 130.dp)
+                    .padding(bottom = size(FluentGlobalTokens.SizeTokens.Size160)),
+                basicCardTokens = if (tokenName == stringResource(id = R.string.shadow_tokens)) previewShadowToken(
+                    tokensList[selectedToken] as FluentGlobalTokens.ShadowTokens
+                )
+                else if (tokenName == stringResource(id = R.string.corner_radius_tokens)) previewCornerRadiusToken(
+                    tokensList[selectedToken] as FluentGlobalTokens.CornerRadiusTokens
+                )
+                else if (tokenName == stringResource(id = R.string.stroke_width_tokens)) previewStrokeWidthToken(
+                    tokensList[selectedToken] as FluentGlobalTokens.StrokeWidthTokens
+                ) else null
+            ) {
+                Box(contentAlignment = Alignment.Center, modifier = Modifier.fillMaxSize()) {
+                    SetPreviewContent(tokenName, tokensList[selectedToken])
+                }
+            }
+
+            PillTabs(metadataList = tabsList, scrollable = true, selectedIndex = selectedToken)
```

Comment:
Add bringIntoViewRequester modifier & configure it to bring the dropdown content in main screen automatically when previewContent goes out of the screen & needed to scroll to bring it in visible region